newlib/libm/test/math.c

```diff
@@ -233,7 +233,7 @@ int calc;
 extern int reduce;
 
 
-frontline (FILE *f,
```

Comment:
looks like this file puts the type on the line before the function; if you do that, then you can remove some of the changes which move the type.

Reply:
Ah, the other files had them in front of the function, and I ended up with a mixed change - I then just put them on the same line using a regex to have "homogenous" change. But I can keep it consistent and adapt it tomorrow.

Reply:
yeah, sorry the code base has such a range of formatting conventions. I'm hesitant to go through and make it pretty as that will make merging the few changes that are made to newlib harder.

Looks like we're close to having your changes merged; just wanted to thank you for all of this good work!

Reply:
I gave the formatting some thought, and I essentially came to the same conclusion: A reformat will make merging changes from/to upstream newlibc a huge pain. On the other hand, it would make the code more maintainable, as there would be no need to care for a file's "custom" formatting. The best option would be to convince the newlib people to algorithmically apply some consistent formatting to all files, and then apply that here as well.

Thank /you/ for so promptly and kindly merging all my changes :) I think we're about 80% done here: I still have some hacks in local branches (though these will be obsolete in the future). With these the tests are now building, but some fail (probably some printf & float issues, I will need to investigate this further - but release duties call). Once that's all sorted out, I'll try to see if we can somehow put a "ready to go" CompCert bundle somewhere, so that the CI can also test against CompCert as well. Also, my HiFive1 arrived, so I've got some real riscv silicon to play around with.